### DISCUSSION TITLE
What's the difference between `scrape_timeout` and `scrape_timeout_offset`?

### DISCUSSION THREAD

**Discussion by Avihais12344** [burningalchemist/sql_exporter#641] [Q&A] [answered]
I am pretty new to this project (and `Prometheus` in general) , and I couldn't find what's the difference between `scrape_timeout` and `scrape_timeout_offset`?

**Comment by burningalchemist** [comment, marked as answer]
Hi @Avihais12344, I guess you can check this file for some comments:
https://github.com/burningalchemist/sql_exporter/blob/master/documentation/sql_exporter.yml

**Reply by Avihais12344** [reply]
Yea, that explain it, thanks.

But why isn't it at the chart docs? As I haven't jumped too much into the code, but to the chart to deploy it...

**Reply by burningalchemist** [reply]
The only reason I have is that the helm chart is not a primary distribution and is not my main focus, so quality-wise it's community/best effort at the moment.

I have plans to revisit the docs significantly to have a proper documentation website with snippets and Q&A, and stuff. But sadly I have limited time, as I usually work on the project outside of my working hours. So things take time. 🤷‍♂️

**Reply by Avihais12344** [reply]
Understandable, maybe I would have time to fix it myself, and help you out :)
